finetune/target_models/masked_language_model.py

```diff
@@ -96,6 +98,34 @@ def __init__(self, *args, **kwargs):
     def _get_input_pipeline(self):
         return MaskedLanguageModelPipeline(self.config)
 
+    def predict_top_k_report(self, input_text, k=5, context=None, **kwargs):
```

Comment:
It should be commented that this input_text is a string and does not accept batches of text.